gpflow/optimizers/scipy.py

```diff
@@ -8,7 +8,7 @@
 __all__ = ["Scipy"]
 
 Loss = tf.Tensor
-Variables = Tuple[tf.Variable]
+Variables = Iterable[tf.Variable]
```

Comment:
```suggestion
Variables = Sequence[tf.Variable]
```
They're ordered right? I believe an `Iterable` doesn't have to be ordered, for example a set